partner_invoicing_mode/tests/test_invoice_mode.py

```diff
@@ -6,7 +6,8 @@
 from freezegun import freeze_time
 
 from odoo import fields
-from odoo.tests.common import Form, TransactionCase
+from odoo.tests import Form
+from odoo.tests.common import TransactionCase
```

Comment:
Why CommonPartnerInvoicingMove isn't depending on TransactionCase?
It implements `setUpClass` and calls `super()` which no parent class implements.

Reply:
I have updated `CommonPartnerInvoicingMove` depending on `TransactionCase`

Reply:
I would ask first the author of this: cc @rousseldenis your opinion on this change?
This `CommonPartnerInvoicingMove` could be implemented this way to use it as a mixin by reverse dependencies, not as a common test class.